eng/Version-Increment.ps1

```diff
@@ -0,0 +1,59 @@
+<#
+.SYNOPSIS
+Bumps up package versions after release
+
+.DESCRIPTION
+This script bumps up package versions following conventions defined at https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#incrementing-after-release-net
+
+.PARAMETER PackageCsprojPath
+Link to the package csproj
+
+.EXAMPLE
+Updating package version for core
+
+Version-Increment.ps1 -PackageCsprojPath "C:\Git\azure-sdk-for-net\sdk\core\Azure.Core\src\Azure.Core.csproj"
+
+#>
+
+[CmdletBinding()]
+Param (
+    [Parameter(Position=0)]
+    [ValidateNotNullOrEmpty()]
+    [string] $PackageCsprojPath
+)
+
+$VERSION_REGEX = "^(?<major>\d+)(\.(?<minor>\d+))?(\.(?<patch>\d+))?(\-(?<pre>[0-9A-Za-z\-\.]+))?(\+(?<build>[0-9A-Za-z\-\.]+))?$"
```

Comment:
Can you please use the exact regex from the semver spect `^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$`  And add a comment pointing to the spec at https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string

Reply:
Used